src/assignments/launch-checklist.rst

```diff
@@ -0,0 +1,118 @@
+Assignment #5: Launch Checklist Form
+====================================
+
+Using our knowledge of forms, the DOM, and HTTP, our favorite space shuttle program has asked us to create a quick launch checklist.
```

Comment:
This is not a big thing, but this sentence assigns sentience to our space shuttle program.  Rather than creating Skynet, how about, "Using our knowledge of forms, the DOM, and HTTP, let's create a quick launch checklist for our favorite space shuttle program."

Reply:
Fixed! Thanks!